tasks/deploy/all.ts

```diff
@@ -0,0 +1,47 @@
+import { task, types } from "hardhat/config";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { DeployFactoryParams } from "./factory";
```

Comment:
It looks like a few of these imports aren't used in this file. I might be missing something, but it would be good to get something like eslint set up to lint the deployment script. I'm fine with doing that in a follow-up FWIW, but this made me think about it.